src/main/resources/db/migration/V1_0__reference_data.sql

```diff
@@ -50,8 +49,15 @@ CREATE TABLE IF NOT EXISTS oasys_question_mapping
     FOREIGN KEY (question_schema_uuid) REFERENCES question_schema (question_schema_uuid)
 );
 
-ALTER TABLE question_schema
-    ADD FOREIGN KEY (reference_data_target) REFERENCES question_schema (question_schema_uuid);
+CREATE TABLE IF NOT EXISTS reference_data_target_mapping
```

Comment:
would it make sense to name this oasys_reference_data_target_mapping?

Reply:
💯 - Updated the table name